packages/client/src/index.ts

```diff
@@ -26,11 +26,12 @@ export default class PerfectumClient {
                 logging,
                 sendMetricsUrl,
                 sendMetricsData,
-                sendMetricsCallback
+                sendMetricsCallback,
+                maxPaintTime = Number.POSITIVE_INFINITY,
```

Comment:
I think we need set default 60 seconds. Because now after this changes we need manual change and i think we will not break any system

Reply:
Fixed it.